src/OrchardCore.Modules/OrchardCore.Localization/Manifest.cs

```diff
@@ -22,3 +22,11 @@
     Dependencies = new[] { "OrchardCore.Localization" },
     Category = "Internationalization"
 )]
+
+[assembly: Feature(
+    Id = "OrchardCore.Localization.CulturePicker",
+    Name = "Culture Picker",
```

Comment:
Do reflect that this is for the admin in the name. Perhaps `CulturePicker.Admin`?

Reply:
I was adding the shape in the admin folder but it throws an exception, frankly I didn't think about changing the feature Id, the description is fine IMHO. Otherwise we should do similar thing for `ContentCulturePicker`

Reply:
The functionality is about adding a culture picker for the admin. The feature's name is just "Culture Picker". As a user, when enabling a feature called "Culture Picker" I wouldn't expect a culture picker to only appear on the admin.

Reply:
This implies to rename the feature name too to something like `Admin Culture Picker`

Reply:
Yes.